### PR TITLE
rm: faq: add information on extending the kernel boot args for lmp-base

### PR DESCRIPTION
**Pull request by vanmaegima**
Signed-off-by: Vanessa Maegima <vanessa.maegima@foundries.io>

### CODE REVIEW

**Comment by doanac** [comment]
Docs for b5fcc546df0bb0873b96667bd5fc6d812faf8230 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1143/docs/artifacts/html/index.html

**Review by ricardosalveti — APPROVED** [review]
Nice, LGTM.

File: source/reference-manual/troubleshooting/troubleshooting.rst [hunk 1]
```diff
@@ -184,6 +184,25 @@ Make sure you set the correct ``<machine>`` and other variables as needed.
...
+
+**meta-subscriber-overrides/recipes-bsp/u-boot/u-boot-base-scr.bbappend:**
+
+.. prompt:: text
+
+    FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+
+.. note::
+    If testing a reference board supported in ``meta-lmp``, the original ``uEnv.txt.in``
+    file can be found in ``meta-lmp/meta-lmp-bsp/recipes-bsp/u-boot/u-boot-base-scr/<board>/uEnv.txt.in``.
```
Comment (angolini) [inline source/reference-manual/troubleshooting/troubleshooting.rst:204]:
```suggestion
    file can be found in ``meta-lmp/meta-lmp-bsp/recipes-bsp/u-boot/u-boot-base-scr/<machine>/uEnv.txt.in``.
```

File: source/reference-manual/troubleshooting/troubleshooting.rst [hunk 2]
```diff
@@ -184,6 +184,25 @@ Make sure you set the correct ``<machine>`` and other variables as needed.
     device. It is not necessarily needed on devices specifying the partition
     path directly with ``root=``.
 
+Now if ``DISTRO=lmp-base`` is set, the kernel command line can be extended by
```
Comment (angolini) [inline source/reference-manual/troubleshooting/troubleshooting.rst:187]:
```suggestion
Now, if ``DISTRO=lmp-base`` is set, the kernel command line can be extended by
```

**Review by angolini — APPROVED** [review]
Minor fixes, other than that LGTM

**Comment by doanac** [comment]
Docs for 9ecafd9618f82cc7f5881060d17a0336cae63b89 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1145/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 6a275dd967cda503ed205b79193b6632fbdd8f01 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1146/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 48a19c230130edfab384cb7ab09eb8d73c7e4256 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1148/docs/artifacts/html/index.html

**Review by Tim-Anderson — APPROVED** [review]
LGTM

**Pull request merged**